src/Confluent.Kafka/.nuspec

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0"?>
+<package>
+    <metadata>
+    <id>MapR.Streams.Dotnet</id>
```

Comment:
you're suggesting Confluent develop a library and then brand it with a competitors name?